src/upnp_cm.cc

```diff
@@ -138,32 +138,3 @@ void ConnectionManagerService::processSubscriptionRequest(const std::unique_ptr<
     ixmlDocument_free(event);
 #endif
 }
-
-void ConnectionManagerService::sendSubscriptionUpdate(const std::string& sourceProtocolCsv)
```

Comment:
I'd love to keep that. It looks like there is some UPnP request we do not implement yet

Reply:
Kept.